### PR TITLE
286 zi profile

### LINKED ISSUE #286: zi's profile
## Description
add zi profile 

## Plan
- [ ] add name
- [ ] add three words
- [ ] add description
- [ ] add image
- [ ] add social media link

## Acceptance Criteria  
- [ ] on the tribe page i can see zi profile with image, three word and description add social media link

### PR DESCRIPTION
**Pull request by katjam**
Fixes #286

## Description

- Zi profile
-

### CODE REVIEW

**Comment by zii-67** [comment]
> This looks great. One check - does the llama image license say we can use it freely without credit?

checked and changed to a new licensed image.

**Comment by Nikomus** [comment]
@zii-67 I can't see any more changes to the image - did you push the new one?

**Comment by zii-67** [comment]
sorry, I just pushed in like 1 min ago.

File: src/data/tribers/zi.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+words:
+  - Curious
+  - Learner
+  - Dreamer
+image: images/zi.jpg
+social:
+  github: 'zii-67'
+---
+
+Start joining Neontribe in 2019. Currently, I'm a computer science degree student who's doing my placement year. Enjoying coding with everyone and exploring Exeter at the same time. Having a head start in experiencing my working life after graduate and it's suprisingly more fun than what I am expecting. Am enthusiastic to come to work everyday, to see and pet the poddle which it's owner is working behind me, learn new things as well.
```
Comment (marxian) [inline src/data/tribers/zi.md]: poodle ;-)

Comment (zii-67) [reply src/data/tribers/zi.md]: Alright. Done correcting :D Thank you.

**Pull request merged**